packages/common/src/bls/CacheDecoderInteractor.ts

```diff
@@ -144,13 +144,13 @@ export class CacheDecoderInteractor {
    */
   async compressRelayRequest (_: { relayRequest: RelayRequest, cachedEncodedData?: PrefixedHexString }): Promise<RelayRequestCachingResult> {
     const nonce = toBN(_.relayRequest.request.nonce)
-    const paymaster = toBN(_.relayRequest.relayData.paymaster)
 
     // TODO: return the tuple of 'address, type' so that all addresses can be queried in one RPC request!
-    const addressesCompressed = await this.compressAddressesToIds([
+    const resolved = await this.compressAddressesToIds(
```

Comment:
Are you sure this is better as 4 parameters than with an array of arrays? 
Maybe we should do ```function foo ([a, b, c, d]: string[4]): void {}```?